Add Crust into dStorage section

badkk opened the pull request.

DannyS03 commented on docs/develop/crusthelpers.md:
```diff
@@ -0,0 +1,26 @@
+
+The Crust protocol is 100% compatible with the [IPFS](https://ipfs.io/) protocol, and it matches people who have hard drive space to spare with those who need to store data or host content. Crust supports most contract platforms, such as Polygon/Solana/NEAR/etc. with its cross-chain solution.
+
+> To learn more about Crust, check its [Decentralized Storage market](https://wiki.crust.network/docs/en/DSM) and [Cross-chain Storage Solution](https://wiki.crust.network/docs/en/buildCrossChainSolution#ii-native-ipfs-integration). Also, you can start building with [Crust Build-101](https://wiki.crust.network/docs/en/build101)
```
```suggestion
:::info Learn more about Crust
Check out the [Decentralized Storage market](https://wiki.crust.network/docs/en/DSM) and [Cross-chain Storage Solution](https://wiki.crust.network/docs/en/buildCrossChainSolution#ii-native-ipfs-integration) documentation. You can also start building with Crust Storage by following the [Crust Build-101](https://wiki.crust.network/docs/en/build101) guide.
:::
```

DannyS03 commented on docs/develop/crusthelpers.md:
```diff
@@ -0,0 +1,26 @@
+image: https://matic.network/banners/matic-network-16x9.png
+---
+
+[Crust](https://crust.network) provides a decentralized storage network for Web3. It is designed to realize core values of decentralization, privacy and assurance. Crust supports multiple storage-layer protocols such as IPFS and exposes instant accessible on-chain storage functions to users.
```
```suggestion
[Crust Network](https://crust.network) provides a decentralized storage network for web3. It is designed to realize the core values of decentralization, privacy and assurance. Crust supports multiple storage-layer protocols such as IPFS, and exposes instantly accessible on-chain storage functions to users.
```

DannyS03 commented on docs/develop/crusthelpers.md:
```diff
@@ -0,0 +1,26 @@
+
+[Crust](https://crust.network) provides a decentralized storage network for Web3. It is designed to realize core values of decentralization, privacy and assurance. Crust supports multiple storage-layer protocols such as IPFS and exposes instant accessible on-chain storage functions to users.
+
+The Crust protocol is 100% compatible with the [IPFS](https://ipfs.io/) protocol, and it matches people who have hard drive space to spare with those who need to store data or host content. Crust supports most contract platforms, such as Polygon/Solana/NEAR/etc. with its cross-chain solution.
```
```suggestion
The protocol matches people who have hard drive space to spare with those who need to store data or host content. Crust supports most smart contract platforms with its cross-chain solution, including Polygon. Find out all the platforms that Crust storage supports on their [official website](https://crust.network/).
```

DannyS03 commented on docs/develop/crusthelpers.md:
```diff
@@ -0,0 +1,26 @@
+
+## Crust Storage Helpers
+
+- [Crust Files](https://crustfiles.io): the 1st web3 personal cloud storage build by Crust community, supports multi-web3 wallet login and provide fully decentralized and privacy storage function for all web3 users.
```
```suggestion
- [Crust Files](https://crustfiles.io): Built by the Crust community, Crust Files is first web3 personal cloud storage that supports multi-web3 wallet login and provides fully decentralized and privacy storage function for all web3 users.
```

DannyS03 commented on docs/develop/crusthelpers.md:
```diff
@@ -0,0 +1,26 @@
+## Crust Storage Helpers
+
+- [Crust Files](https://crustfiles.io): the 1st web3 personal cloud storage build by Crust community, supports multi-web3 wallet login and provide fully decentralized and privacy storage function for all web3 users.
+- [IPFS W3Auth Gateway](https://docs.ipfs.tech/concepts/ipfs-gateway/#authenticated-gateways): Web3 authenticated IPFS gateway, support all the IPFS *WRITE* functions, that helps users upload file or host content after pass the web3 auth verification.
```
```suggestion
- [IPFS W3Auth Gateway](https://docs.ipfs.tech/concepts/ipfs-gateway/#authenticated-gateways): A web3 authenticated IPFS gateway that supports all the IPFS *WRITE* functions and helps users upload files or host content after passing the W3Auth verification.
```

DannyS03 commented on docs/develop/crusthelpers.md:
```diff
@@ -0,0 +1,26 @@
+
+- [Crust Files](https://crustfiles.io): the 1st web3 personal cloud storage build by Crust community, supports multi-web3 wallet login and provide fully decentralized and privacy storage function for all web3 users.
+- [IPFS W3Auth Gateway](https://docs.ipfs.tech/concepts/ipfs-gateway/#authenticated-gateways): Web3 authenticated IPFS gateway, support all the IPFS *WRITE* functions, that helps users upload file or host content after pass the web3 auth verification.
+- [IPFS W3Auth Pinning Service](https://wiki.crust.network/docs/en/buildIPFSW3AuthPin): A standard IPFS pinning service with web3 authentication, it allows users access Crust storage resources without interacting with Crust blockchain.
```
```suggestion
- [IPFS W3Auth Pinning Service](https://wiki.crust.network/docs/en/buildIPFSW3AuthPin): A standard IPFS pinning service with web3 authentication that allows users to access Crust storage resources without needing to interact with the Crust blockchain.
```

DannyS03 commented on docs/develop/crusthelpers.md:
```diff
@@ -0,0 +1,26 @@
+- [Crust Files](https://crustfiles.io): the 1st web3 personal cloud storage build by Crust community, supports multi-web3 wallet login and provide fully decentralized and privacy storage function for all web3 users.
+- [IPFS W3Auth Gateway](https://docs.ipfs.tech/concepts/ipfs-gateway/#authenticated-gateways): Web3 authenticated IPFS gateway, support all the IPFS *WRITE* functions, that helps users upload file or host content after pass the web3 auth verification.
+- [IPFS W3Auth Pinning Service](https://wiki.crust.network/docs/en/buildIPFSW3AuthPin): A standard IPFS pinning service with web3 authentication, it allows users access Crust storage resources without interacting with Crust blockchain.
+- [Crust Github Action](https://github.com/marketplace/actions/crust-ipfs-pin): A standard github action which can involve the fully decentralized hosting CI/CD with Crust.
```
```suggestion
- [Crust Github Action](https://github.com/marketplace/actions/crust-ipfs-pin): A standard GitHub action which can include fully decentralized CI/CD hosting with Crust.
```

DannyS03 commented on docs/home/blockchain-basics/basics-consensus-mechanism.md:
```diff
@@ -23,7 +23,7 @@ Proof-of-Stake mechanism achieve consensus by requiring users to stake an amount
 
 4. PoSpace: Proof of Space <br></br>
-This kind of consensus mechanism is useful in decentralized file storage applications like storj.io, filecoin, where nodes prove they have legitimate capacity in their hardware. Sometimes also referred as PoStorage or PoCapacity.
+This kind of consensus mechanism is useful in decentralized file storage applications like storj.io, filecoin, crust, where nodes prove they have legitimate capacity in their hardware. Sometimes also referred as PoStorage or PoCapacity.
```
```suggestion
This kind of consensus mechanism is useful in decentralized file storage applications like in Storj.io, Filecoin, and Crust, where nodes prove they have legitimate capacity in their hardware. Sometimes also referred as PoStorage or PoCapacity.
```

DannyS03 reviewed (changes requested):
thank you for the contribution @badkk! left some suggestions. Some of the other pages will likely be updated soon too

badkk commented:
> thank you for the contribution @badkk! left some suggestions. Some of the other pages will likely be updated soon too

Thank you so much for your comments @DannyS03 ! I've already changed and updated, please take a look!

DannyS03 commented on docs/develop/crusthelpers.md:
```diff
@@ -11,16 +11,17 @@ keywords:
-> To learn more about Crust, check its [Decentralized Storage market](https://wiki.crust.network/docs/en/DSM) and [Cross-chain Storage Solution](https://wiki.crust.network/docs/en/buildCrossChainSolution#ii-native-ipfs-integration). Also, you can start building with [Crust Build-101](https://wiki.crust.network/docs/en/build101)
+:::info Learn more about Crust
+Check its [Decentralized Storage market](https://wiki.crust.network/docs/en/DSM) and [Cross-chain Storage Solution](https://wiki.crust.network/docs/en/buildCrossChainSolution#ii-native-ipfs-integration). Also, you can start building with [Crust Build-101](https://wiki.crust.network/docs/en/build101)
 
```
```suggestion
:::
```

DannyS03 commented on docs/develop/crusthelpers.md:
```diff
@@ -11,16 +11,17 @@ keywords:
 
-> To learn more about Crust, check its [Decentralized Storage market](https://wiki.crust.network/docs/en/DSM) and [Cross-chain Storage Solution](https://wiki.crust.network/docs/en/buildCrossChainSolution#ii-native-ipfs-integration). Also, you can start building with [Crust Build-101](https://wiki.crust.network/docs/en/build101)
+:::info Learn more about Crust
+Check its [Decentralized Storage market](https://wiki.crust.network/docs/en/DSM) and [Cross-chain Storage Solution](https://wiki.crust.network/docs/en/buildCrossChainSolution#ii-native-ipfs-integration). Also, you can start building with [Crust Build-101](https://wiki.crust.network/docs/en/build101)
```
```suggestion
Check out the [Decentralized Storage market](https://wiki.crust.network/docs/en/DSM) and [Cross-chain Storage Solution](https://wiki.crust.network/docs/en/buildCrossChainSolution#ii-native-ipfs-integration) offered by Crust Network. Also, you can start building with [Crust Build-101](https://wiki.crust.network/docs/en/build101).
```

DannyS03 commented on docs/develop/crusthelpers.md:
```diff
@@ -11,16 +11,17 @@ keywords:
-- [IPFS W3Auth Gateway](https://docs.ipfs.tech/concepts/ipfs-gateway/#authenticated-gateways): Web3 authenticated IPFS gateway, support all the IPFS *WRITE* functions, that helps users upload file or host content after pass the web3 auth verification.
-- [IPFS W3Auth Pinning Service](https://wiki.crust.network/docs/en/buildIPFSW3AuthPin): A standard IPFS pinning service with web3 authentication, it allows users access Crust storage resources without interacting with Crust blockchain.
-- [Crust Github Action](https://github.com/marketplace/actions/crust-ipfs-pin): A standard github action which can involve the fully decentralized hosting CI/CD with Crust.
+- [Crust Files](https://crustfiles.io): Built by the Crust community, Crust Files is first web3 personal cloud storage that supports multi-web3 wallet login and provides fully decentralized and privacy storage function for all web3 users.
```
```suggestion
- [Crust Files](https://crustfiles.io): Built by the Crust community, Crust Files is the first personal cloud storage for web3 that supports multi-web3 wallet login and provides fully decentralized and privacy storage function for all users.
```

DannyS03 reviewed (changes requested):
thanks @badkk! Just a few more minor suggestions

badkk commented:
> thanks @badkk! Just a few more minor suggestions

Sure! @DannyS03 Already solved!

DannyS03 commented:
cheers @badkk
